src/components/editor/components/RecipientInput.scss

```diff
@@ -1,82 +1,123 @@
 @import "../../../res/styles/_required";
 
-.recipients-input tags-input .tags .tag-item {
-  font-family: $font-family-base;
-  font-weight: 500;
-  background-color: theme-color-level("primary", $alert-bg-level);
-  border-color: theme-color-level("primary", $alert-border-level);
-  transition: background-color 0.3s, border-color 0.3s;
-}
-
-@mixin tag-variant($background, $border, $color) {
-  color: $color;
-  @include gradient-bg($background);
-  border-color: $border;
-}
-
-.recipients-input tags-input .tags .remove-button {
-  color: inherit;
-}
-
-@each $color, $value in $theme-colors {
-  .recipients-input tags-input .tags .tag-#{$color} {
-    @include alert-variant(theme-color-level($color, $alert-bg-level), theme-color-level($color, $alert-border-level), theme-color($color));
+@import "/node_modules/bootstrap/scss/badge";
+@import "/node_modules/bootstrap/scss/dropdown";
+@import "/node_modules/bootstrap/scss/forms";
+@import "/node_modules/bootstrap/scss/utilities/display";
+@import "/node_modules/bootstrap/scss/utilities/flex";
+@import "/node_modules/bootstrap/scss/utilities/spacing";
+
+// Get spacer value from bootstrap
+$sp-1: map-get($spacers, 1);
+
+.recipients-input {
+  @extend .form-control, .mb-1;
+  padding: 0 $sp-1 $sp-1 0; // mirror to tags and input margins
+  position: relative; // for the dropdown
+  height: initial; // input should be stretchable
+
+  &:focus-within {
+    @extend .form-control:focus;
+  }
+
+  .tag-selected-list {
+    /* The layout has to be stretchable because the input should take all the space left */
+    @extend .d-flex, .flex-wrap;
+
+    .tag-wrapper {
+      @extend .badge; // we later copy $badge-padding-y in .tag-input-wrapper
+      margin: $sp-1 0 0 $sp-1; // mirror to .recipients-input padding
+
+      .tag-remove {
+        /* Reset button styles */
+        padding: 0;
+        border: none;
+        background: none;
+        font-weight: 700;
+        font-size: 16px;
+        margin-left: 8px;
+      }
+
+      &::before {
+        font: normal normal normal 14px/1 'MailvelopeIcons';
+        line-height: 1;
+
+        /* Better Font Rendering =========== */
```

Comment:
I think these are not required anymore if you, as we do, only target newer browser versions.

Reply:
Removed